BioMed.Api/BioMed.Api/Extensions/DatabaseSeeder.cs

```diff
@@ -0,0 +1,752 @@
+
+            foreach (var visit in visits)
+            {
+                payments.Add(new Payment()
+                {
+                    Amount = visit.TotalPrice + (visit.TotalPrice * (decimal)0.025),
+                    Date = _faker.Date.Between(DateTime.Now.AddYears(-2), DateTime.Now),
+                    VisitId = visit.Id,
+                });
+            }
+
+            context.Payments.AddRange(payments);
+            context.SaveChanges();
+        }
+
+        public static void CreateAnalysisTypes(BioMedDbContext context)
+        {
+            if (context.AnalysisTypes.Any()) return;
+
+            var analysisTypes = new List<AnalysisType>
+            {
+                new AnalysisType() { Name = "Complete Blood Count (CBC)" },
+                new AnalysisType() { Name = "Blood Chemistry Panel" },
+                new AnalysisType() { Name = "Blood Type and Rh Factor" },
+                new AnalysisType() { Name = "Coagulation Panel (Clotting Factors)" },
+                new AnalysisType() { Name = "Lipid Panel (Cholesterol Levels)" },
+                new AnalysisType() { Name = "Blood Glucose (Blood Sugar)" },
+                new AnalysisType() { Name = "Routine Urinalysis" },
+                new AnalysisType() { Name = "Microscopic Examination of Urine" },
+                new AnalysisType() { Name = "X-rays" },
+                new AnalysisType() { Name = "Computed Tomography (CT) Scan" },
+                new AnalysisType() { Name = "Magnetic Resonance Imaging (MRI)" },
+                new AnalysisType() { Name = "Ultrasound" },
+                new AnalysisType() { Name = "Positron Emission Tomography (PET) Scan" },
+                new AnalysisType() { Name = "Mammography" },
+                new AnalysisType() { Name = "Electrocardiogram (ECG or EKG)" },
+                new AnalysisType() { Name = "Electroencephalogram (EEG)" },
+                new AnalysisType() { Name = "Upper Gastrointestinal Endoscopy" },
+                new AnalysisType() { Name = "Colonoscopy" },
+                new AnalysisType() { Name = "Bronchoscopy" },
+                new AnalysisType() { Name = "Biopsy" },
+                new AnalysisType() { Name = "Pap Smear" },
+                new AnalysisType() { Name = "Genetic Testing" },
+                new AnalysisType() { Name = "Allergy Testing" },
+                new AnalysisType() { Name = "Thyroid Function Tests" },
+                new AnalysisType() { Name = "Hormone Levels (e.g., estrogen, testosterone)" },
+                new AnalysisType() { Name = "Clinical pathology" },
+                new AnalysisType() { Name = "Hematology" },
+                new AnalysisType() { Name = "Microbiology" },
+                new AnalysisType() { Name = "Clinical chemistry" },
+            };
+
+            context.AnalysisTypes.AddRange(analysisTypes);
+            context.SaveChanges();
+        }
+
+        public static void CreateLaboratoryResults(BioMedDbContext context)
```

Comment:
It would be more efficient to convert these public methods to private methods as well.

Reply:
why do you think so?

Reply:
Because these methods are used only in this class and cannot be called in other classes.

Reply:
It`s clear, i correct my mistakes
Thank you